test-tests/shared/src/test/scala/zio/test/mock/ExpectationSpec.scala

```diff
@@ -301,8 +301,8 @@ object ExpectationSpec extends ZIOBaseSpec {
         equalTo(
           UnmetExpectationsException(
             List(
-              Module.command -> equalTo(2),
-              Module.command -> equalTo(3)
+              Module.command -> (equalTo(2): Assertion[Any]),
```

Comment:
Why is it necessary to explicitly pass `Assertion[Any]` here? And why `Any`?

Reply:
After looking at this more it appears that a couple of the mock method type signatures were incorrect. The assertion is an assertion about the input, not the output. Good catch!